resources/ig-expansion-parameters.json

```diff
@@ -1 +1,72 @@
-{"resourceType":"ImplementationGuide","id":"ig","extension":[{"url":"http://hl7.org/fhir/StructureDefinition/igpublisher-folder-resource","valueString":"C:\\work\\org.hl7.fhir.igs\\bulk-data\\resources"},{"url":"http://hl7.org/fhir/StructureDefinition/igpublisher-folder-pages","valueString":"C:\\work\\org.hl7.fhir.igs\\bulk-data\\spec"}],"url":"http://hl7.org/fhir/us/bulkdata/ImplementationGuide/ig","version":"0.1.0","name":"BulkDataAccess","title":"FHIR Bulk Data Access","status":"active","experimental":false,"publisher":"FHIR Project","packageId":"hl7.fhir.us.bulkdata","license":"CC0-1.0","fhirVersion":["4.0.0","4.0.0"],"definition":{"resource":[{"reference":{"reference":"OperationDefinition/export"},"name":"Definition for $export"},{"reference":{"reference":"OperationDefinition/group-export"},"name":"Definition for Group $export"},{"reference":{"reference":"OperationDefinition/patient-export"},"name":"Definition for Patient $export"}],"page":{"nameUrl":"index.html","title":"FHIR Bulk Data Access","generation":"html"},"parameter":[{"code":"apply","value":"version"},{"code":"generate-json","value":"true"},{"code":"generate-xml","value":"true"},{"code":"generate-turtle","value":"true"}]}}
\ No newline at end of file
+{
+  "resourceType": "ImplementationGuide",
+  "id": "ig",
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/igpublisher-folder-resource",
+      "valueString": "C:\\work\\org.hl7.fhir.igs\\bulk-data\\resources"
```

Comment:
These look unxpected -- local dev windows paths?

Reply:
@grahamegrieve can you provide some color into whether or not this is needed? I see it in other ig-expansion-param files as well